.travis.yml

```diff
@@ -43,9 +43,9 @@ env:
 
     matrix:
         # Make sure that egg_info works without dependencies
-        - PYTHON_VERSION=3.4 SETUP_CMD='egg_info'
         - PYTHON_VERSION=3.5 SETUP_CMD='egg_info'
         - PYTHON_VERSION=3.6 SETUP_CMD='egg_info'
+        - PYTHON_VERSION=3.7 SETUP_CMD='egg_info'
```

Comment:
you can drop all but one of the egg_info builds, too.

Reply:
Which one should I keep? 3.7?

Reply:
It shouldn't matter, the one you like the most :)